src/pages/api-docs/index.js

```diff
@@ -27,7 +27,7 @@ function APIDocs() {
                  To learn about GraphQL queries generally, the GraphQL Foundation has helpful resources.`)}
                 <ul>
                     <li>
-                        <a href="https://api.tarkov.dev/___graphql" target="_blank" rel="noopener noreferrer">{t('Tarkov.dev GraphQL playground')}</a>
+                        <a href="https://api.tarkov.dev/graphql" target="_blank" rel="noopener noreferrer">{t('Tarkov.dev GraphQL playground')}</a>
```

Comment:
This new one doesn't seem to work

Reply:
That's because I changed it to the API URL instead of the new playground URL. Fixed now.